src/_locales/de/messages.json

```diff
@@ -31,30 +47,42 @@
         "message": "Erkennen von unbekannten Titeln erzwingen",
         "description": "Option title"
     },
-    "optionScrobblePodcasts": {
-        "message": "Scrobble podcasts",
+    "optionScrobbleRecognizedTracks": {
+        "message": "Scrobble recognized or edited tracks (default)",
         "description": "Option label"
     },
-    "optionScrobblePodcastsTitle": {
-        "message": "Scrobble podcast episodes",
+    "optionScrobbleRecognizedTracksTitle": {
+        "message": "Only scrobble tracks if they are recognized or edited",
+        "description": "Option title"
+    },
+    "optionScrobbleEditedTracksOnly": {
+        "message": "Scrobble edited tracks only",
+        "description": "Option label"
+    },
+    "optionScrobbleEditedTracksOnlyTitle": {
+        "message": "Only scrobble tracks if they are edited",
         "description": "Option title"
     },
     "optionsHidden": {
-        "message": "Versteckt",
+        "message": "Hidden",
         "description": "'Hidden' section"
     },
     "optionScrobblePercent": {
-        "message": "Scrobble bei",
+        "message": "Scrobble at",
         "description": "Option title"
     },
     "optionScrobblePercentSuffix": {
-        "message": "der Songlänge",
+        "message": "of song duration",
         "description": "Option title"
     },
     "optionYtDesc": {
-        "message": "Wenn du beide Optionen deaktivierst, wird die Erweiterung YouTube videos aus allen Kategorien scrobblen.",
+        "message": "If you turn off both options, the extension will scrobble YouTube videos from all categories.",
```

Comment:
It seems like it's killing a lot of existing translations (just one example). Is this not in transifex?

Reply:
There is an `--mode` flag which has `onlytranslated` but that leaves the `message` field empty :/ 

```
     "optionScrobbleEditedTracksOnlyTitle": {
-        "message": "Only scrobble tracks if they are edited",
+        "message": "",
         "description": "Option title"
     },
```

Reply:
Exactly! Feels like it would be more problematic :/

Reply:
Updated with script as you suggested